src/Sylius/Bundle/AttributeBundle/AttributeType/CheckboxAttributeType.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Bundle\AttributeBundle\AttributeType;
+
+use Sylius\Component\Attribute\AttributeType\AttributeTypeInterface;
+use Sylius\Component\Attribute\Model\AttributeValueInterface;
+use Symfony\Component\Validator\ConstraintViolationList;
+use Symfony\Component\Validator\Context\ExecutionContextInterface;
+
+/**
+ * @author Mateusz Zalewski <mateusz.zalewski@lakion.com>
+ */
+class CheckboxAttributeType implements AttributeTypeInterface
+{
+    const TYPE = 'checkbox';
+
+    /**
+     * @return string
+     */
+    public function getStorageType()
```

Comment:
`getStorageType` and `getType` should have docblocks inherited from AttributeTypeInterface. Same for rest of `*AttributeType`